apps/api/src/app/workflows-v2/usecases/delete-workflow/delete-workflow.usecase.ts

```diff
@@ -0,0 +1,63 @@
+import { BadRequestException, Injectable } from '@nestjs/common';
+
+import { EnvironmentRepository, NotificationTemplateRepository } from '@novu/dal';
+import { DeleteWorkflowCommand } from './delete-workflow.command';
+import { ControlValuesRepository, MessageTemplateRepository } from '@novu/dal/src';
+import { WorkflowNotFoundException } from '../../exceptions/workflow-not-found';
+
+@Injectable()
+export class DeleteWorkflowUseCase {
+  constructor(
+    private notificationTemplateRepository: NotificationTemplateRepository,
+    private environmentRepository: EnvironmentRepository,
+    private msgTemplateRepo: MessageTemplateRepository,
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Please use the full camel-cased name, it ensures that lookups (i.e. for find/replace) across the codebase are consistent.‏
```suggestion
    private messageTemplateRepository: MessageTemplateRepository,
```

Reply:
Agree here the number of times i searched in the code for something like `messageTemplateRepository.find` is countless, i will update it.